samtranslator/model/api/api_generator.py

```diff
@@ -208,6 +216,116 @@ def _add_cors(self):
         # Assign the Swagger back to template
         self.definition_body = editor.swagger
 
+    def _add_auth(self):
+        """
+        Add Auth configuration to the Swagger file, if necessary
+        """
+
+        if not self.auth:
+            return
+
+        INVALID_ERROR = "Invalid value for 'Auth' property"
```

Comment:
Why constify this if you're only going to use it once? It seems like the rest of the error messages are just string literals

Reply:
Yeah I considered changing this but it's how it is in other parts of the code so I was being consistent